docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -169,6 +169,36 @@ the file /etc/sawtooth/validator.toml.
 
       $ sudo systemctl restart sawtooth-validator
 
+Sawtooth Rest API Configuration
```

Comment:
REST API should be all upper-case here and in the text below. (I didn't comment on them all.)